src/viking.js

```diff
@@ -1,11 +1,75 @@
+//Samuel Ledeboer
+
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+
+    this.attack = function () {
+        return this.strength;
+    }
+    
+    /*this.receiveDamage = function (damage) {
+        this.health -= damage;
+    } 
+    */
+}
+
+Soldier.prototype.receiveDamage = function (damage) {
+    this.health -= damage;
+}
+
+
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+    this.name = name;
+    //this.health = health;
+    //this.strength = strength;
+    
+    Soldier.call(this, health, strength)
+}
+
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
+
+//recieve damage
+Viking.prototype.receiveDamage = function (damage) {
+    this.health-= damage;
+    if (this.health > 0) {
+        return `${this.name} has received ${damage} points of damage`
+    } else {
+        return `${this.name} has died in act of combat`
+    }
+}
 
 // Saxon
-function Saxon() {}
+function Saxon( health, strength ) {
+    this.health = health;
+    this.strength = strength;
+ }
+ Saxon.prototype = Object.create(Soldier.prototype);
+ Saxon.prototype.constructor = Saxon;
+ 
+
 
+//attack function
+ Saxon.prototype.attack = function() {
+    return this.strength;
+ };
```

Comment:
The saxon.attack already exists because you call the prototype of the soldier line 52